install/k3d/multi-cluster/README.md

```
@@ -429,12 +429,12 @@ kubectl --context k3d-openchoreo-cp create secret generic observabilityplane-def
 
 ### 10. Configure DataPlane to use default ObservabilityPlane (Optional)
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)